webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModal.js

```diff
@@ -0,0 +1,65 @@
+import React, { createContext } from 'react';
+import { Modal } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import Header from './subcomponents/ForemanModalHeader';
+import Footer from './subcomponents/ForemanModalFooter';
+// import { translate as __ } from '../../common/I18n';
+
+export const ModalContext = createContext(null);
+
+const ForemanModal = props => {
+  const { isOpen, onClose, title } = props;
+  if (!isOpen) return null;
```

Comment:
I am not sure if that's the reason,
but when closing the modal it seem that everything disappear too fast..
I also checked it with PF3 where there's a fade-out feeling.

Reply:
@laviro Good point, if React isn't rendering then there's no way for the transition animations to happen.  I'll take a look at hiding the modal another way.

Reply:
@laviro Thanks, removed that line and now the transition is smooth :)